controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -119,7 +83,49 @@ func CreateIqeJobResource(cache *rc.ObjectCache, cji *crd.ClowdJobInvocation, en
 		TerminationMessagePolicy: core.TerminationMessageReadFile,
 	}
 
+	return c
+}
+
+func createSeleniumContainer(name string, nn types.NamespacedName, cji *crd.ClowdJobInvocation, env *crd.ClowdEnvironment, app *crd.ClowdApp) core.Container {
+	// create env vars
+	envVars := []core.EnvVar{
+		{Name: "HOME", Value: "/home/selenium"},
```

Comment:
https://github.com/RedHatQE/selenium-images/blob/master/standalone/Dockerfile#L26

Reply:
Nice, will remove that